website/docs/02 - Connection.md

```diff
@@ -136,29 +149,65 @@ Using the connection instance, you can access the database server's current date
 ### Syntax
 
 ```ts
-getDbDate(): Promise<string>
+getDbDate(options?: GetDbDateOptions): Promise<string>
 ```
 
+### Parameters
+
+| Parameter | Type     | Description                                              | Example |
+| --------- | -------- | -------------------------------------------------------- | ------- |
+| `options` | `object` | [Options object](#options-object-properties) (see below) |         |
```

Comment:
With the links you need to be careful if the same heading appears in the document multiple times.  It won't go to the correct subheading.  If you look at the "Reading by Id" document you should be able to see how the link gets tweaked there to bring to the correct subheading.

Side note, if you use the "Open markdown preview" option in vscode you can test the links to make sure they bring you to the right place.

Reply:
Links fixed.

Reply:
Looks good!  Resolving this conversation.